### ISSUE TITLE
No skins?

### ISSUE THREAD

**Issue by Ariamiro** [huytd/agar.io-clone#50]
skins can do the game more fun

**Comment by IgorAntun** [comment]
Yeah we could do that, but it's like **really** low priority right now, unless someone here want to work on it :)

**Comment by Ariamiro** [comment]
why not a todo list?

**Comment by IgorAntun** [comment]
We could have a TODO.md, yes. Good idea.

**Comment by fry-** [comment]
If somebody adds this someday, I can help with making the graphics :stuck_out_tongue:

**Comment by huytd** [comment]
We have the wiki page, just add it to [Coming Features Page](https://github.com/huytd/agar.io-clone/wiki/Coming-Features)

**Comment by IgorAntun** [comment]
![image](https://cloud.githubusercontent.com/assets/5935946/7927025/bcb417d2-08b0-11e5-9f82-55c9481bb039.png)
Full list of known user skins @ Agar.IO (may be outdated or whatever)

**Comment by fry-** [comment]
I think this resource is more complete and has better graphics:
http://www.agarioguide.com/skins/
We can recreate them all, but also could add our own or change them to what we like more (change name and / or look). :3
They all look like sloppy drawn in some pixel graphic program. If you want to preserve this style, would be fine for me, I could do it in Gimp. But if you want more clean shaped graphics, I would like to make them in Inkscape (and then render it to whatever image format and size you need). :blush:

**Comment by IgorAntun** [comment]
Yeah, in my opinion I think I'd prefer something more clean shaped, yes, that's something to be discussed :)
As of image format/size, what about .svg so it scales properly?

**Comment by fry-** [comment]
Yes, with Inkscape it would be an .svg :smile: but I don't know if this can be used directly in the game, I have no clue about the technical backgroud :stuck_out_tongue: But once the vector image is done (and you keep the .svg of course) you can render it to any format (.png for example) with any size/scale. Pixel graphics (Gimp) wouldn't allow this of course, once the image would be done, it's final and up- / downscaling would result in ugly blur.
So I basically prefer Inkscape and it's vectorgraphics too :stuck_out_tongue:
